build-llvm.py

```diff
@@ -168,17 +168,21 @@ def parse_parameters(root_folder):
     parser.add_argument("--lto",
                         metavar="LTO_TYPE",
                         help=textwrap.dedent("""\
-                        Build the final compiler with either full LTO (full) or ThinLTO (thin), which can
+                        Build the final compiler with either ThinLTO (thin) or  full LTO (full), which can
                         improve compile time performance.
 
+                        Only use full LTO if you have more than 64 GB of memory. ThinLTO uses way less memory,
+                        compiles faster because it is fully multithreaded, and it has almost identical
+                        performance (within 1%% usually) to full LTO.
```

Comment:
```suggestion
                        performance (within 1% usually) to full LTO.
```

unintended?

Reply:
Needed to avoid this:

```
$ ./build-llvm.py -h
Traceback (most recent call last):
  File "./build-llvm.py", line 998, in <module>
    main()
  File "./build-llvm.py", line 974, in main
    args = parse_parameters(root_folder)
  File "./build-llvm.py", line 280, in parse_parameters                                                                                                  return parser.parse_args()
  File "/usr/lib/python3.8/argparse.py", line 1780, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/usr/lib/python3.8/argparse.py", line 1812, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)                                                                                          File "/usr/lib/python3.8/argparse.py", line 2018, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/usr/lib/python3.8/argparse.py", line 1958, in consume_optional
    take_action(action, args, option_string)
  File "/usr/lib/python3.8/argparse.py", line 1886, in take_action
    action(self, namespace, argument_values, option_string)
  File "/usr/lib/python3.8/argparse.py", line 1056, in __call__
    parser.print_help()
  File "/usr/lib/python3.8/argparse.py", line 2506, in print_help
    self._print_message(self.format_help(), file)
  File "/usr/lib/python3.8/argparse.py", line 2490, in format_help
    return formatter.format_help()
  File "/usr/lib/python3.8/argparse.py", line 294, in format_help
    help = self._root_section.format_help()
  File "/usr/lib/python3.8/argparse.py", line 225, in format_help                                                                                        item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.8/argparse.py", line 225, in <listcomp>
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.8/argparse.py", line 225, in format_help
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.8/argparse.py", line 225, in <listcomp>
    item_help = join([func(*args) for func, args in self.items])                                                                                       File "/usr/lib/python3.8/argparse.py", line 541, in _format_action
    help_text = self._expand_help(action)
  File "/usr/lib/python3.8/argparse.py", line 633, in _expand_help                                                                                       return self._get_help_string(action) % params
TypeError: %u format: a number is required, not dict
```

Reply:
Ah fun.